### PR TITLE
HBW-200 Highlight opened task in table

### PR DESCRIPTION
**Pull request by vladdoncheg**

### CODE REVIEW

File: hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx [hunk 1]
```diff
@@ -104,11 +105,17 @@ const withTaskListContext = (WrappedComponent) => {
     };
 
     openTask = (index) => {
-      this.setState(prevState => ({ activeTask: prevState.tasks[index] }));
+      this.setState(prevState => ({
+        activeTask: prevState.tasks[index],
+        opened:     index,
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx]: I think it's a bit confusing to use index or list item for extremely similar purposes. Let's use either index or full item in both cases.

**Comment by vladdoncheg** [comment]
Скриншот 2019-10-18 16 21 14
Скриншот 2019-10-18 16 21 22

File: hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx (hunk 1 above)
Comment (TimAle) [reply hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx]: Same about naming: we have `chosen`, `opened` and `activeTask`. Either all with postfix `Task` or all without.
More say, don't you think that `opened` and `activeTask` are the same things?

Comment (vladdoncheg) [reply hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx]: Yep, you're right. I thought about it but something seemed strange in my mind 😕

**Pull request merged**